cmd/docker_commands.go

```diff
@@ -98,10 +100,10 @@ func RunCommandAndListen(commandValue string, args []string, logger appsodylogge
 			}
 		}()
 
-		err = execCmd.Start()
+		out, err := SeperateOutput(execCmd)
```

Comment:
Hey Luke. The new tests look great. This change however, I don't think should be made. The `cmd.Start()` func behaves different from the `cmd.Output()` which you are replacing. Lets leave `Start()`'s alone for now. This may be the reason your travis build is failing.

Reply:
I did think this would be the case but didn't want to write it off immediately! I'll remove the change & the corresponding test and leave just the DockerInspect tests.

Reply:
Yay travis passed! :-) So were you able to test this with any of the kubectl functions? Reason I ask is because this issue was originally reported against a kubectl apply command. It would be good to verify the original issue is fixed.

Reply:
Yes I tried manually with all of the Kube functions I could. I'll add an automated test to verify this, though!